package com.example.online_school.exception.errorMessage;

public class ErrorMessage {

    public static final String ID_NOT_FOUND = "This id was not found";

    public static final String USER_ALREADY_EXISTS = "The user already exists";

    public static final String ROLE_ALREADY_EXISTS = "The role already exists";

    public static final String ROLES_NOT_FOUND = "Roles not found";

    public static final String ROLE_ID_NOT_FOUND = "This role id was not found";

    public static final String USERS_WITH_ROLE_EXIST = "Users with this role exist";

    public static final String NULL_POINTER = "Null pointer";

    public static final String USER_WITH_SUCH_EMAIL_EXISTS = "The user with such a mail exists";

    public static final String USERNAME_ALREADY_EXISTS = "The username already exists";

    public static final String INVALID_EMAIL = "The email is invalid";

    public static final String INVALID_PASSWORD = "The password is invalid";

    public static final String INVALID_PHONE_NUMBER = "The phone number is invalid";

    public static final String INVALID_LASTNAME = "The lastname is invalid";

    public static final String INVALID_FIRST_NAME = "The first name is invalid";

    public static final String INVALID_USERNAME = "The username is invalid";
}
